### PR TITLE
fix: add intents in example

### PR DESCRIPTION
**Pull request by sus2790**
## About

This pull request is about (X), which does (Y).

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues) (If existent):.
  - resolves #
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

**Comment by Damego** [comment]
message content intent doesn't included in DEFAULT

**Comment by sus2790** [comment]
TBR...

File: examples/bot.py [hunk 1]
```diff
@@ -10,7 +10,10 @@
 # When you make a bot, we refer to it as the "client."
 # The client is the main object that interacts with the Gateway, what talks to Discord.
 # The client is also the main object that interacts with the API, what makes requests with Discord.
-client = interactions.Client("your bot token will go here.")
+client = interactions.Client(
+    "your bot token will go here.",
+    intents=interactions.Intents.DEFAULT | interactions.Intents.GUILD_MESSAGE_CONTENT,
```
Comment (i0bs) [inline examples/bot.py:15]: Can we make this just the default?

Would also add comments

**Review by i0bs — CHANGES_REQUESTED** [review]

File: examples/bot.py (hunk 1 above)
Comment (Damego) [reply examples/bot.py:15]: > Can we make this just the default?

Example bot has `on_message_create` event so we need message intent for this

Comment (i0bs) [reply examples/bot.py:15]: Would rather have an example just to show bare connection tbh, other example files should be considered

Comment (Catalyst4222) [reply examples/bot.py:15]: I agree, a separate example about intents should be added

Comment (sus2790) [reply examples/bot.py:15]: Wait...., why it's `on_ready` not `on_start`?

Comment (mAxYoLo01) [reply examples/bot.py:15]: > Wait...., why it's `on_ready` not `on_start`?

The gateway can reconnect multiple times during the bot's life, and `on_ready` is triggered for each reconnection. However, `on_start` is triggered only **once** when the bot has been started for the first time, so it's better to use it.

Comment (sus2790) [reply examples/bot.py:15]: Dead.

Comment (sus2790) [reply examples/bot.py:15]: is time to resolving conversation

Comment (i0bs) [reply examples/bot.py:15]: Thoughts haven't changed: people would rather have a separate example page for intents, which would mean gutting out the `on_message_create` event handler code in total here.

**Comment by Catalyst4222** [comment]
Can we get an update on this? A guide on intent would be very appreciated, though having them in the first example could be confusing to users

**Comment by EdVraz** [comment]
Closing in 3 days if there is no more activity.

**Pull request closed without merging**